### PR TITLE
Making FAQ more mentor specific

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <bmulligan@linuxfoundation.org>

### CODE REVIEW

**Comment by dealako** [comment]
Assigned to @pranab-bajpai and @dibyaranjanlf for review.

**Review by dibyaranjanlf — COMMENTED** [review]
Hi @xmulligan 
Thanks for the PR for the documentation updates.  After reviewing the changes with the product owner, I have some points to keep. Can you update your PR to include the following changes, going chronologically : 

1. How much time is expected from Mentors to commit to mentoring? _(this sentence sounds better from writing perspective)_
2. How long are mentors engaged in the program? _("are" to be used as we are referring mentors in plural)_
3. Also, if you can please fix the DCO sign-off in the format `Signed-off-by: John Doe <john.doe@gmail.com>`

File: mentorship/mentor-guide/mentorship-faqs.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Mentorship FAQs
 
-**How much time mentors expect to commit to mentoring?**
+**How much time are mentors expect to commit to mentoring?**
```
Comment (xmulligan) [inline mentorship/mentor-guide/mentorship-faqs.md]:
```suggestion
**How much time is expected from Mentors to commit to mentoring?**
```

File: mentorship/mentor-guide/mentorship-faqs.md [hunk 2]
```diff
@@ -12,13 +12,9 @@ Absolutely! For mentees who successfully graduate, Linux Foundation promotes the
 
 The Linux Foundation offers mentorship programs around the year with specific term dates. Please check the projects of your interest to find details of the program term dates.
 
-**How long is a mentee engaged in the program?**
+**How long is a mentors engaged in the program?**
```
Comment (xmulligan) [inline mentorship/mentor-guide/mentorship-faqs.md]:
```suggestion
**How long are mentors engaged in the program?**
```

**Comment by xmulligan** [comment]
@dibyaranjangithub thanks for the feedback. all of the suggestions have been added

**Review by dibyaranjanlf — APPROVED** [review]
@xmulligan Thanks for implementing the proposed changes.

@dealako Please merge the changes.

**Pull request merged**